artfiles.py

```diff
@@ -5,63 +5,73 @@
 # Copyright (c) 2016, InnoGames GmbH
 #
 
-import time
-import sys
-import urllib
-import argparse
-from socket import gethostname
+from time import time
+from urllib import urlopen
+from argparse import ArgumentParser
 
-parser = argparse.ArgumentParser()
-parser.add_argument(
-    "-u",
-    "--user",
-    dest="http_user",
-    type=str,
-    required=True,
-    help="the http user name to authenticate")
-parser.add_argument(
-    "-p",
-    "--pass",
-    dest="http_pass",
-    type=str,
-    required=True,
-    help="the http password to authenticate")
-args = parser.parse_args()
 
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "-u",
+        "--user",
+        dest="http_user",
+        type=str,
+        required=True,
+        help="the http user name to authenticate")
+    parser.add_argument(
+        "-p",
+        "--pass",
+        dest="http_pass",
+        type=str,
+        required=True,
+        help="the http password to authenticate")
+    return parser.parse_args()
 
-artfiles_url = 'https://%s:%s@dcp.c.artfiles.de/api/stats/get_estats.html' % (
-    args.http_user, args.http_pass)
-data = urllib.urlopen(artfiles_url)
-csv = data.readlines()
 
-now = str(int(time.time()))
+def main():
+    args = parse_args()
+    artfiles_url = (
+        'https://{}:{}@dcp.c.artfiles.de/api/stats/get_estats.html'
+        .format(args.http_user, args.http_pass)
+    )
+    data = urlopen(artfiles_url)
+    now = str(int(time()))
+    for csv in data.readlines():
+        if csv.startswith('"level3"') or csv.startswith('"w408'):
+            parse_and_print(csv, now)
 
-graphite = ""
-for line in csv:
-    if line.startswith('"level3"') or line.startswith('"w408'):
-        dc, rack, pdu_nr, maxcur, measurementtype, maxval_watt, curval = line.split(
-            ',')
-        dc = dc.strip('"').replace('/', '_')
-        rack = rack.strip('"').translate(None, '/')
-        pdu_nr = pdu_nr.strip('"')
 
-        if maxcur == '"16A"':
-            graphite += 'powerline.' + dc + '.' + rack + '.' + \
-                pdu_nr + '.max ' + '16.00 ' + str(now) + '\n'
-        elif maxcur == '"10A"':
-            graphite += 'powerline.' + dc + '.' + rack + '.' + \
-                pdu_nr + '.max ' + '10.00 ' + str(now) + '\n'
-        elif maxcur == '"redundant"':
-            graphite += 'powerline.' + dc + '.' + rack + '.' + \
-                pdu_nr + '.max ' + '20.00 ' + str(now) + '\n'
+def parse_and_print(csv, now):
+    values = [v.strip('\n "') for v in csv.split(',')]
```

Comment:
I wonder if it is really safe to do `strip('\n "'`) for every single value. Before not every value was stripped that way. I see you mentioned the double quotes below, and it might be that everything is ok. But that is something I would test. ^^